File: test/aidbox_sdk/snapshots/typescripthl7-fhir-r4-core/Flag.ts
import { Period } from "./Period";
import { CodeableConcept } from "./CodeableConcept";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type Flag = DomainResource & {
    code: CodeableConcept;
    author?: Reference;
    period?: Period;
    status: string;
    subject: Reference;
    category?: CodeableConcept[];
    encounter?: Reference;
    identifier?: Identifier[];
};

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/Flag.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import Period
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class Flag(DomainResource):
    code: CodeableConcept
    author: Optional[Reference] = None
    period: Optional[Period] = None
    status: str
    subject: Reference
    category: Optional[List[CodeableConcept]] = field(default_factory=list)
    encounter: Optional[Reference] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)